pkg/clusters/addons/kong/kong.go

```diff
@@ -29,14 +30,34 @@ const (
 	AddonName clusters.AddonName = "kong"
 
 	httpPort = 80
+
+	// DefaultEnterpriseImageRepo default kong enterprise image
+	DefaultEnterpriseImageRepo = "kong/kong-gateway"
+
+	// DefaultEnterpriseImageTag latest kong enterprise image tag
+	DefaultEnterpriseImageTag = "2.5.0.0-alpine"
+
+	// KongEnterpriseTestLicense is the kong license data secret name
+	KongEnterpriseTestLicense = "kong-enterprise-license"
+
+	// EnterpriseAdminPasswordSecretName is the kong admin seed password
+	EnterpriseAdminPasswordSecretName = "kong-enterprise-superuser-password"
+
+	// EnterpriseKongAdminDefaultPWD is kong admin password
+	EnterpriseKongAdminDefaultPWD = "password"
```

Comment:
We should not provide a default password of "password" even if this is a testing utility as we should not make such a strong assumption about how this utility will be used, and because it's trivial to generate a password if one isn't provided by the caller explicitly.

```suggestion
```

Reply:
removed 36696c21a8f868ac2930664b0a8eb06bee14ac2f